src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
-body {
+@import './utils/variable';
+@import './blocks/stopwatch';
```

Comment:
Make sure the file path in the import statement is correct and the file exists. The convention is to use underscores for SCSS partials and to omit the file extension, but this depends on the project setup.